taker-frontend/src/App.tsx

```diff
@@ -157,49 +193,46 @@ export const App = () => {
                     <VStack>
                         {connectionStatus}
                         <Routes>
-                            <Route
-                                path="/wallet"
-                                element={<>
-                                    <Wallet walletInfo={walletInfo} />
-                                </>}
-                            />
-
-                            <Route
-                                path="/"
-                                element={<>
-                                    <VStack divider={<StackDivider borderColor="gray.500" />} spacing={4}>
+                            <Route path="/">
+                                <Route
+                                    path="wallet"
+                                    element={<>
+                                        <Wallet walletInfo={walletInfo} />
+                                    </>}
+                                />
+                                <Route
+                                    path="long"
+                                    element={<VStack>
+                                        <NavigationButtons/>
                                         <Trade
                                             longOrder={longOrder}
                                             shortOrder={shortOrder}
```

Comment:
Why would we pass long and short order into the `Trade` component if that component is now either long or short? I think it should just be order now...

Reply:
This is outdated, sorry for the confusion in the commits.